pkg/ebpfev/event_interface.go

```diff
@@ -0,0 +1,14 @@
+package ebpfev
```

Comment:
[Required]: This package should be nested under `v1`

Reply:
done